front-end-node/NodeInspectorOverrides.js

```diff
@@ -58,6 +60,18 @@ WebInspector.NodeInspectorOverrides.prototype = {
       showMainAppFile,
       null
     );
+  },
+  
+  _mergeConnectionQueryParams: function(){
+    var RuntimeQueryParamsObject = Runtime._queryParamsObject;
+    if (RuntimeQueryParamsObject['ws'] && RuntimeQueryParamsObject['port']){
+      if (RuntimeQueryParamsObject['ws'].indexOf('?') === -1){
```

Comment:
I don't know but some plugins could extend it, right? I would add it just to be safe... ;)